src/backend/expungeservice/parsers/param_parser.py

```diff
@@ -0,0 +1,51 @@
+from html.parser import HTMLParser
+
+
+class ParamParser(HTMLParser):
+
+    def __init__(self):
+        HTMLParser.__init__(self)
+        self.event_target = ''
+        self.event_argument = ''
+        self.view_state = ''
+        self.view_state_generator = ''
+        self.event_validation = ''
+        self.node_id = ''
+
+    def handle_starttag(self, tag, attrs):
+        if tag == 'input':
+            switcher = {
+                '__EVENTTARGET': self.__event_target,
+                '__EVENTARGUMENT': self.__event_argument,
+                '__VIEWSTATE': self.__view_state,
+                '__VIEWSTATEGENERATOR': self.__view_state_generator,
+                '__EVENTVALIDATION': self.__event_validation,
+                'NodeID': self.__node_id
+            }
+            switcher.get(dict(attrs)['name'], self.__default)(dict(attrs).get('value'))
```

Comment:
What will these parameters be used for? It would be nice to add some documentation.

Reply:
Good point, I should definitely add some documentation. They're actually explained what they're for in issue #17 and a little bit in the git commit. I will update with documentation.

Reply:
Sorry, I didn't see the commit message. So these parameters needed when sending the search query?

Reply:
Exactly. Need to navigate to the page and grab these parameters in order to post a valid search request. It seems ASP.NET does that to verify the request originates from controls rendered by their server.